moviefind/src/js/components/Movie.js

```diff
@@ -1,28 +1,30 @@
-var React =require('react');
+var React = require('react');
 var AppActions = require('../actions/AppActions');
```

Comment:
It's great to see the use of named imports for React and Component, making the code more readable. However, for consistency, consider using named imports for `AppActions` and `AppStore` as well.